forgerock-authenticator/src/main/java/org/forgerock/android/authenticator/Push.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2020 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.authenticator;
+
+import org.forgerock.android.authenticator.util.SortedList;
+
+import java.util.List;
+
+/**
+ * Represents an instance of a Push authentication mechanism. Associated with an Account.
+ */
+public class Push extends Mechanism {
+
+    /** The endpoint used for registration */
+    public static final String REG_ENDPOINT_KEY = "r";
```

Comment:
Declare private?

Reply:
This is going to be used by the PushFactory class, so needs to be public.